README.md

```diff
@@ -22,6 +22,12 @@ An implementation of VHD [VHD specification](https://technet.microsoft.com/en-us
     2. https://github.com/Azure/azure-sdk-for-go
             This package is used to make Microsoft Azure storage API calls.
 
+# Features
+
+    1. Fast uploads - This tool offers faster uploads than the [Azure Xplat CLI](https://github.com/Azure/azure-xplat-cli).
```

Comment:
indent was to match styling of the line above. Will remove extra line. Never am sure of the CLI branding, will fix too.